include/cinder/app/Renderer.h

```diff
@@ -78,13 +78,13 @@ class Renderer {
 #if defined( CINDER_COCOA )
 	#if defined( CINDER_MAC )
 		virtual void	setup( CGRect frame, NSView *cinderView, RendererRef sharedRenderer, bool retinaEnabled ) = 0;
-		virtual CGContextRef			getCgContext() { throw; } // the default behavior is failure
 		virtual CGLContextObj			getCglContext() { throw; } // the default behavior is failure
 		virtual CGLPixelFormatObj		getCglPixelFormat() { throw; } // the default behavior is failure
 	#elif defined( CINDER_COCOA_TOUCH )
 		virtual void		setup( const Area &frame, UIView *cinderView, RendererRef sharedRenderer ) = 0;
 		virtual bool		isEaglLayer() const { return false; }
 	#endif
+	virtual CGContextRef	getCgContext() { throw; } // the default behavior is failure
```

Comment:
This function is overridden in `Renderer2d` when building `CINDER_COCOA_TOUCH` so it needed to be moved out of the `CINDER_MAC` exclusive block.